incorporate AC review feedback

igrigorik opened the pull request:
@plehegar @siusin @toddreifsteck @yoavweiss PTAL.

I believe this should capture all of the outstanding feedback and questions from the AC review.

igrigorik committed:
incorporate AC review feedback

siusin commented:
LGTM.

@wseltzer ping? plh is on vacation this week... 

[1] [comments from the AC review](https://www.w3.org/2002/09/wbs/33280/2018-webperf-charter/results).

yoavweiss reviewed (approved):
LGTM

wseltzer commented:
2 comments: 
1. Should the privacy consideration be stated more strongly?  Instead of "The group reserves the right not to publish a specification if acceptable mitigation strategies cannot be found" say "The group *should not* publish a specification if acceptable mitigation strategies cannot be found." 

2. I don't think the decision policy edit addresses the comment, which asks that Calls for Consensus be called out explicitly. Is that something the group might do, for example by putting "call for consensus" into the subject header when sending minutes of meetings that contain resolutions?

igrigorik commented:
Thanks for the feedback Wendy.

> Should the privacy consideration be stated more strongly? Instead of "The group reserves the right not to publish a specification if acceptable mitigation strategies cannot be found" say "The group should not publish a specification if acceptable mitigation strategies cannot be found."

Practically speaking, I think they're stating the same thing, but I'm ok with proposed wording. 

> I don't think the decision policy edit addresses the comment, which asks that Calls for Consensus be called out explicitly. Is that something the group might do, for example by putting "call for consensus" into the subject header when sending minutes of meetings that contain resolutions?

The decision here was to tighten our requirements around publishing minutes, and improving our process to get those out in a timely fashion for everyone to review. I'm happy to follow "[dd/mm/yy] meeting minutes / call for consensus" convention for subject lines carrying link to the minutes.

---

@yoavweiss @toddreifsteck any objections?

igrigorik committed:
update mitigation wording, specify subject line for consensus

igrigorik commented:
Updated to capture the above feedback.

@toddreifsteck @yoavweiss PTAL. If this looks good, I think we're good to merge.

yoavweiss commented on index.html:
```diff
@@ -502,7 +502,7 @@ <h2>
         </h2>
         <p id="public">
-          Technical discussions for this Working Group are conducted in <a href="https://www.w3.org/2015/Process-20150901/#confidentiality-levels">public</a>. Meeting minutes from teleconference and face-to-face meetings will be archived for public review, and technical discussions and issue tracking will be conducted in a manner that can be both read and written to by the general public. Working ...
+          Technical discussions for this Working Group are conducted in <a href="https://www.w3.org/2015/Process-20150901/#confidentiality-levels">public</a>. Meeting minutes from teleconference and face-to-face meetings will be archived for public review (with "[dd/mm/yy] meeting minutes / call for consensus" subject line), and technical discussions and issue tracking will be conducted in a mann ...
```
I think it'd be better if we add the "call for consensus" bit only when publishing minutes where we're actually calling for consensus (e.g. the group decided to adopt a new proposal from incubation).

So maybe something along the lines of "Meeting minutes from teleconference and face-to-face meetings will be archived for public review. When significant decisions were taken during the teleconference, the minutes will be published while including a 'call for consensus' in their subject line."

igrigorik replied:
Good call. Also, realized that I was updating the wrong section. Pushed an update, PTAL.

igrigorik committed:
call for consensus

- revert earlier wording
- update async decision making section

yoavweiss commented:
LGTM. Good to merge from my perspective

igrigorik commented:
Thanks Yoav. 

@toddreifsteck I'll let you do the last pass and merge if this looks reasonable. :)

toddreifsteck commented:
LGTM. Merging!
